src/app/modules/OpenshiftStreams/components/StreamsTableConnected/useDeleteDialog.ts

```diff
@@ -2,19 +2,31 @@ import { KafkaInstancesProps } from "@rhoas/app-services-ui-components";
 import { KafkaInstanceEnhanced } from "./useKafkaInstances";
 import { useCallback } from "react";
 import { ModalType, useModal } from "@rhoas/app-services-ui-shared";
+import useChrome from "@redhat-cloud-services/frontend-components/useChrome";
 
 export function useDeleteDialog() {
+  const { analytics } = useChrome();
   const { showModal: showDeleteModal } =
     useModal<ModalType.KasDeleteInstance>();
   const openDeleteDialog: KafkaInstancesProps<KafkaInstanceEnhanced>["onDelete"] =
     useCallback(
       (instance, onDone) => {
+        analytics.track("RHOSAK Delete Kafka instance", {
```

Comment:
Above you don't have _Kafka_ in the name. I think it is not needed, as RHOSAK is already there